node/src/network.rs

```diff
@@ -213,6 +223,29 @@ impl<const N: usize> crate::Network for Kadcast<N> {
         Ok(())
     }
 
+    /// Broadcast a GetResource request.
+    ///
+    /// By utilizing the randomly selected peers per bucket in Kadcast, this
+    /// broadcast does follow the so-called "Flood with Random Walk" blind
+    /// search (resource discovery).
+    ///
+    /// A receiver of this message is supposed to either look up and return the
```

Comment:
Just a nitpick: this should be: `is supposed to look up the resource and either return it or, if not found, rebroadcast the message to the next bucket`

The way it's written now seems like the lookup part is conditional, which is not the case.

Reply:
Noted.